src/com/sap/piper/analytics/Analytics.groovy

```diff
@@ -0,0 +1,106 @@
+package com.sap.piper.analytics
+
+import com.cloudbees.groovy.cps.NonCPS
+import org.jenkinsci.plugins.workflow.steps.MissingContextVariableException
+
+class Analytics implements Serializable{
```

Comment:
other name suggestions welcome

Reply:
maybe call it `Telemetry` instead. This rather fits the intended purpose ...